src/SW_Model.c

```diff
@@ -113,7 +113,7 @@ void SW_MDL_read(SW_MODEL* SW_Model, char *InFiles[], LOG_INFO* LogInfo) {
 	FILE *f;
 	int y, cnt;
 	TimeInt d;
-	char *p, enddyval[6], errstr[MAX_ERROR], *MyFileName, inbuf[MAX_FILENAMESIZE];
+	char *p, enddyval[6], errstr[MAX_LOG_SIZE], *MyFileName, inbuf[MAX_FILENAMESIZE];

```

Comment:
see earlier comment about `MAX_LOG_SIZE` - I think we should create local `errstr[]` of `SW_MDL_read()` to be shorter than `MAX_LOG_SIZE`